.release-it.json

```diff
@@ -0,0 +1,19 @@
+{
+  "plugins": {
+    "release-it-lerna-changelog": {
+      "infile": "CHANGELOG.md"
+    },
+    "release-it-yarn-workspaces": {
+      "publish": false
+    }
+  },
+  "git": {
+    "commitMessage": "v${version}"
+  },
```

Comment:
I would recommend https://github.com/simplabs/qunit-dom/blob/master/.release-it.js#L11 to match what we're currently doing here

Reply:
Our tags so far have been without the `v` 😬 So I was maintaining that, but I'm fine with changing it.

Reply:
😱 sorry, nevermind then 😅